docs/networking/cilium.md

```diff
@@ -138,6 +138,16 @@ Once the secret has been created, encryption can be enabled by setting `enableEn
       enableEncryption: true
 ```
 
+#### Custom images
+
+`docker.io/cilium/cilium` it's the default image but there're cases, after Docker Hub T&C changes, where requirements needs another registry. You can set you own setting `image`, `operatorImage` or, `hubbleRelayImage`:
```

Comment:
I would not refer to the Docker TOS here.

Reply:
Will remove, no problem